src/frontend/packages/core/src/components/design-system/Formik/SubmitButton/FormikSubmitButton.tsx

```diff
@@ -59,18 +62,22 @@ export const FormikSubmitButton: FunctionComponent<FormikSubmitButtonProps> = ({
       {showTooltipMessage && (
         <Tip content={getTooltipMessage()}>
           <Box>
-            <Button {...props} primary disabled={true} />
+            <Button {...props} disabled={true}>
+              {props.label}
+            </Button>
           </Box>
         </Tip>
       )}
       {canSubmit && (
-        <Button
+        <LoadingButton
           {...props}
-          primary
+          color={'primary'}
```

Comment:
As you did it in previous commit
```suggestion
          color="primary"
```